src/main/kotlin/domain/person/Person.kt

```diff
@@ -1,18 +1,33 @@
 package domain.person
 
 import domain.card.Card
-import domain.card.Cards
-import domain.constant.GameState
+import domain.card.Hand
+import domain.money.Money
+import domain.result.Score
+import domain.state.Finished
+import domain.state.Started
+import domain.state.State
 
-abstract class Person(open val name: String) {
-    val cards = Cards()
+abstract class Person(val name: String) {
 
-    fun receiveCard(vararg card: Card) {
-        card.forEach { cards.add(it) }
-        checkState()
+    abstract var state: State
```

Comment:
```suggestion
    abstract val state: State
```
부모 클래스에서 val을 사용해도, 자식 클래스에서 var로 오버라이딩이 가능해요.

Reply:
val 을 var로 오버라이딩 가능한 것은 처음 알았습니다..! 다만 현재 부모 클래스 메서드에서 var 이 필요하므로 유지하겠습니다!

알려주셔서 감사합니다ㅎㅎ

Reply:
이 부분에 대한 부연설명이 없었네요.
만약 부모 클래스에서도 객체를 생성해서 state를 변경한다면, var을 사용하시는 것이 좋습니다.
부모 클래스에서는 state를 변경하지 않고 자식 클래스에서만 state를 변경한다면, 부모에서는 val을 사용하고 자식에서는 var로 오버라이딩해서 사용하는 것을 다른 개발자들에게도 유도할 수 있는 장점이 있습니다. 😅 
하지만 지금 구조에서는 부모 클래스에서도 stay 함수를 통해 state를 변경하고 있으므로, var을 사용하는 것이 좋겠네요. 👍